Unable to validate openapi on Fedora

Running the xtask like this: `cargo xtask validate-openapi`, results in:

```
    Finished `dev` profile [unoptimized + debuginfo] target(s) in 13.40s
     Running `target/debug/xtask validate-openapi`
Writing openapi.yaml to "/tmp/openapi.yaml"
Validating spec (/src/openapi.yaml)
[main] WARN  io.swagger.v3.parser.OpenAPIV3Parser - Exception while reading:
io.swagger.v3.parser.exception.ReadContentException: Unable to read location `/src/openapi.yaml`
        at io.swagger.v3.parser.OpenAPIV3Parser.readContentFromLocation(OpenAPIV3Parser.java:310)
        at io.swagger.v3.parser.OpenAPIV3Parser.readLocation(OpenAPIV3Parser.java:95)
        at io.swagger.parser.OpenAPIParser.readLocation(OpenAPIParser.java:16)
        at org.openapitools.codegen.cmd.Validate.execute(Validate.java:52)
        at org.openapitools.codegen.cmd.OpenApiGeneratorCommand.run(OpenApiGeneratorCommand.java:32)
        at org.openapitools.codegen.OpenAPIGenerator.main(OpenAPIGenerator.java:66)
Caused by: java.lang.RuntimeException: Could not find /src/openapi.yaml on the classpath
        at io.swagger.v3.parser.util.ClasspathHelper.loadFileFromClasspath(ClasspathHelper.java:33)
        at io.swagger.v3.parser.OpenAPIV3Parser.readContentFromLocation(OpenAPIV3Parser.java:301)
        ... 5 common frames omitted
[main] ERROR i.s.parser.SwaggerCompatConverter - failed to read resource listing
java.lang.RuntimeException: Could not find /src/openapi.yaml on the classpath
        at io.swagger.parser.util.ClasspathHelper.loadFileFromClasspath(ClasspathHelper.java:31)
        at io.swagger.parser.SwaggerCompatConverter.readResourceListing(SwaggerCompatConverter.java:206)
        at io.swagger.parser.SwaggerCompatConverter.read(SwaggerCompatConverter.java:122)
        at io.swagger.parser.SwaggerCompatConverter.readWithInfo(SwaggerCompatConverter.java:93)
        at io.swagger.parser.SwaggerParser.readWithInfo(SwaggerParser.java:45)
        at io.swagger.v3.parser.converter.SwaggerConverter.readLocation(SwaggerConverter.java:83)
        at io.swagger.parser.OpenAPIParser.readLocation(OpenAPIParser.java:16)
        at org.openapitools.codegen.cmd.Validate.execute(Validate.java:52)
        at org.openapitools.codegen.cmd.OpenApiGeneratorCommand.run(OpenApiGeneratorCommand.java:32)
        at org.openapitools.codegen.OpenAPIGenerator.main(OpenAPIGenerator.java:66)
[main] ERROR i.s.parser.SwaggerCompatConverter - failed to read resource listing
java.lang.RuntimeException: Could not find /src/openapi.yaml on the classpath
        at io.swagger.parser.util.ClasspathHelper.loadFileFromClasspath(ClasspathHelper.java:31)
        at io.swagger.parser.SwaggerCompatConverter.readResourceListing(SwaggerCompatConverter.java:206)
        at io.swagger.parser.SwaggerCompatConverter.read(SwaggerCompatConverter.java:122)
        at io.swagger.parser.SwaggerCompatConverter.readWithInfo(SwaggerCompatConverter.java:93)
        at io.swagger.parser.SwaggerParser.readWithInfo(SwaggerParser.java:45)
        at io.swagger.v3.parser.converter.SwaggerConverter.readLocation(SwaggerConverter.java:83)
        at io.swagger.parser.OpenAPIParser.readLocation(OpenAPIParser.java:16)
        at org.openapitools.codegen.cmd.Validate.execute(Validate.java:52)
        at org.openapitools.codegen.cmd.OpenApiGeneratorCommand.run(OpenApiGeneratorCommand.java:32)
        at org.openapitools.codegen.OpenAPIGenerator.main(OpenAPIGenerator.java:66)
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "java.util.Collection.size()" because "<parameter1>" is null
        at java.base/java.util.HashSet.<init>(Unknown Source)
        at org.openapitools.codegen.cmd.Validate.execute(Validate.java:54)
        at org.openapitools.codegen.cmd.OpenApiGeneratorCommand.run(OpenApiGeneratorCommand.java:32)
        at org.openapitools.codegen.OpenAPIGenerator.main(OpenAPIGenerator.java:66)
Error: Failed to validate openapi.yaml
```

I can add the `:z` option in the case of `podman`, but then I run into this:

```
    Finished `dev` profile [unoptimized + debuginfo] target(s) in 13.38s
     Running `target/debug/xtask validate-openapi`
Writing openapi.yaml to "/tmp/openapi.yaml"
Error: SELinux relabeling of /tmp is not allowed
Error: Failed to validate openapi.yaml
```

At that point, I gave up.

--- a/xtask/src/openapi.rs
+++ b/xtask/src/openapi.rs
@@ -48,6 +48,8 @@ impl Validate {
                 "--rm",
                 "-v",
                 ".:/src",
+                "--security-opt",
+                "label=disable",
                 "docker.io/openapitools/openapi-generator-cli:v7.7.0",
                 "validate",
                 "-i",

